models/mailchimp/mailchimp_sent_to.sql

```diff
@@ -0,0 +1,7 @@
+select distinct
+	list_id, campaign_id, email_id, email_address, status, send_time as sent_date
+from demo_data.mailchimp_sent_to a
+-- add information about when the campaign was sent
+inner join demo_data.mailchimp_campaigns b
```

Comment:
Any given raw data table should only be selected from a single time. This should rely on {{env.schema}}.mailchimp_campaigns.